.drone.star

```diff
@@ -631,81 +1021,137 @@ def phptests(testType):
 					command = 'make test-php-integration'
 
 			for db in params['databases']:
-				keyString = '-' + category if params['includeKeyInMatrixName'] else ''
-				name = '%s%s-php%s-%s' % (testType, keyString, phpVersion, db.replace(":", ""))
-				maxLength = 50
-				nameLength = len(name)
-				if nameLength > maxLength:
-					print("Error: generated phpunit stage name of length", nameLength, "is not supported. The maximum length is " + str(maxLength) + ".", name)
-					errorFound = True
-
-				result = {
-					'kind': 'pipeline',
-					'type': 'docker',
-					'name': name,
-					'workspace' : {
-						'base': '/var/www/owncloud',
-						'path': 'server/apps/%s' % config['app']
-					},
-					'steps':
-						installCore('daily-master-qa', db, False) +
-						installApp(phpVersion) +
-						installExtraApps(phpVersion, params['extraApps']) +
-						setupServerAndApp(phpVersion, params['logLevel']) +
-						setupCeph(params['cephS3']) +
-						setupScality(params['scalityS3']) +
-						params['extraSetup'] +
-					[
-						{
-							'name': '%s-tests' % testType,
+				for externalType in params['externalTypes']:
+					keyString = '-' + category if params['includeKeyInMatrixName'] else ''
+					filesExternalType = externalType if externalType != 'none' else ''
+					name = '%s%s-php%s-%s' % (testType, keyString, phpVersion, db.replace(":", ""))
+					maxLength = 50
+					nameLength = len(name)
+					if nameLength > maxLength:
+						print("Error: generated phpunit stage name of length", nameLength, "is not supported. The maximum length is " + str(maxLength) + ".", name)
+						errorFound = True
+
+					if (filesExternalType == ''):
+						# for the regular unit test runs, the clover coverage results are in a file named like:
+						# autotest-clover-sqlite.xml
```

Comment:
I suspect that you will need to look at what the "standard" oC10 app `phpunit` tests do for naming the coverage result files. The scripts in core were "a bit odd" and had their own special naming system.